### PR TITLE
Css frameworks a redo of the whole CA

### PR DESCRIPTION
**Pull request by KarenJoo**
I started all over again, reinstalling Bootstrap and Sass + a new design, layout, logo, and color palette. Does this meet the requirements for the CA? am I styling correctly according to Bootstrap styling? are the variables correctly imported?

### CODE REVIEW

File: src/scss/index.scss [hunk 1]
```diff
@@ -1 +1,109 @@
+@import "variables";
+@import "../../node_modules/bootstrap/scss/bootstrap.scss";
 
+$enable-shadows: true;
```
Comment (Fermain) [inline src/scss/index.scss]: Lonely variable? Can you move this with the rest?

File: src/scss/index.scss [hunk 2]
```diff
@@ -1 +1,109 @@
...
+
+
+.btn {
+  text-decoration: none;
```
Comment (Fermain) [inline src/scss/index.scss:31]: I'm a bit surprised this is necessary, I think this is a default in BS

File: src/scss/index.scss [hunk 3]
```diff
@@ -1 +1,109 @@
...
+  }
+
+  p {
+    color: $primary;
+  }
+}
+
+/* body styles */
+body {
+  background-color: $primary;
```
Comment (Fermain) [inline src/scss/index.scss]: `.bg-primary`

File: src/scss/index.scss [hunk 4]
```diff
@@ -1 +1,109 @@
...
+$enable-shadows: true;
+
+
+.btn {
+  text-decoration: none;
+}
+
+/* nav bottom */
+.navbar-bottom {
+  background-color: $secondary;
```
Comment (Fermain) [inline src/scss/index.scss]: `.bg-secondary`

File: src/scss/index.scss [hunk 5]
```diff
@@ -1 +1,109 @@
...
+  //image from: https://www.pexels.com/nb-no/bilde/sjarmerende-om-harmoni-ung-4130215/
+  background-image: url("/img/pexels-elina-sazonova-4130215.jpg");
+  background-size: cover;
+  background-repeat: no-repeat;
+}
+
+/* Authentication page */
+
+.navbar-custom {
+  background-color: $primary;
```
Comment (Fermain) [inline src/scss/index.scss]: DRY, `.bg-primary`

File: src/scss/index.scss [hunk 6]
```diff
@@ -1 +1,109 @@
...
+}
+
+.max-width {
+  max-width: 100%;
```
Comment (Fermain) [inline src/scss/index.scss]: `.mw-100`

File: src/scss/index.scss [hunk 7]
```diff
@@ -1 +1,109 @@
...
+}
+
+/* Log in form */
+.form-container {
+  justify-content: center;
```
Comment (Fermain) [inline src/scss/index.scss:78]: `.justify-content-center`

File: src/scss/index.scss [hunk 8]
```diff
@@ -1 +1,109 @@
...
+    }
+
+    label {
+      color: $light;
+    }
+  }
+}
+
+.form-title {
+  color: $primary;
```
Comment (Fermain) [inline src/scss/index.scss]: `.text-primary`

File: src/scss/index.scss [hunk 9]
```diff
@@ -1 +1,109 @@
...
+}
+
+/* Profile page */
+.navbar-profile {
+  background-color: $primary;
```
Comment (Fermain) [inline src/scss/index.scss:105]: `.bg-primary`

File: src/scss/index.scss [hunk 10]
```diff
@@ -1 +1,109 @@
...
+  .container {
+    align-items: center;
```
Comment (Fermain) [inline src/scss/index.scss:107]: `.align-items-center.justify-content-between`

File: src/scss/index.scss [hunk 11]
```diff
@@ -1 +1,109 @@
...
+  }
+}
+
+
+
+/* profile container */
+.profile-container {
+  background-color: $primary;
+  border-radius: 0px 0px 70px 70px;
+  width: 100%;
```
Comment (Fermain) [inline src/scss/index.scss:132]: `.w-100`

File: src/scss/index.scss [hunk 12]
```diff
@@ -1 +1,109 @@
...
+  .navbar-brand {
+    color: $dark;
+  }
+}
+
+
+
+/* profile container */
+.profile-container {
+  background-color: $primary;
```
Comment (Fermain) [inline src/scss/index.scss:129]: `.bg-primary`

File: src/scss/index.scss [hunk 13]
```diff
@@ -1 +1,109 @@
...
+  border-radius: 0px 0px 70px 70px;
+  width: 100%;
+}
+
+/* Styling for the profile section container */
+
+.profile-section {
+  flex-direction: column;
```
Comment (Fermain) [inline src/scss/index.scss:138]: `.flex-column.justify-content-center.align-items-center`

File: src/scss/index.scss [hunk 14]
```diff
@@ -1 +1,109 @@
...
+  img.profile-image {
+    width: 200px;
+    height: 200px;
+    border-radius: 50%;
+    object-fit: cover;
+  }
+
+  h2 {
+    /* Styling for the user name */
+    color: $secondary;
```
Comment (Fermain) [inline src/scss/index.scss:161]: `.color-secondary`

File: src/scss/index.scss [hunk 15]
```diff
@@ -1 +1,109 @@
...
+.profile-section {
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+
+  img.profile-image {
+    width: 200px;
+    height: 200px;
+    border-radius: 50%;
+    object-fit: cover;
```
Comment (Fermain) [inline src/scss/index.scss:146]: https://getbootstrap.com/docs/5.3/utilities/borders/
https://getbootstrap.com/docs/5.3/utilities/object-fit/

**Review by Fermain — CHANGES_REQUESTED** [review]
You will pass with what you have here, but I am pushing for a gold medal. Your SCSS can be smaller still!

File: src/scss/index.scss (hunk 1 above)
Comment (KarenJoo) [reply src/scss/index.scss]: I realized I actually can delete the lonely enable-shadow variable in the index.scss, as I have already enabled Bootstrap shadows in the _variables.scss, please correct me if I'm wrong.

File: src/scss/index.scss (hunk 2 above)
Comment (KarenJoo) [reply src/scss/index.scss:31]: Oh, you are absolutely right!

File: src/scss/index.scss (hunk 3 above)
Comment (KarenJoo) [reply src/scss/index.scss]: Done

File: src/scss/index.scss (hunk 4 above)
Comment (KarenJoo) [reply src/scss/index.scss]: Done

File: src/scss/index.scss (hunk 5 above)
Comment (KarenJoo) [reply src/scss/index.scss]: Deleted this class name as I am using .navbar

File: src/scss/index.scss (hunk 11 above)
Comment (KarenJoo) [reply src/scss/index.scss:132]: Done

File: src/scss/index.scss (hunk 12 above)
Comment (KarenJoo) [reply src/scss/index.scss:129]: Done

File: src/scss/index.scss (hunk 13 above)
Comment (KarenJoo) [reply src/scss/index.scss:138]: Perfect, thanks

File: src/scss/index.scss (hunk 14 above)
Comment (KarenJoo) [reply src/scss/index.scss:161]: Done

File: feed/index.html [hunk 16]
```diff
@@ -12,10 +12,10 @@
     <title>Feed page</title>
   </head>
 
-  <body>
+  <body class="bg-primary">
     <nav class="navbar navbar-expand-lg bg-secondary shadow">
       <div class="container">
-        <a class="navbar-brand" href="#">
+        <a class="navbar-brand" href="../profile/index.html">
```
Comment (Fermain) [inline feed/index.html]:
```suggestion
        <a class="navbar-brand" href="/profile/index.html">
```

This should work, `/` means "from the root of your project"

Comment (Fermain) [reply feed/index.html]:
```suggestion
        <a class="navbar-brand" href="/profile/">
```

Even better :)

File: src/scss/index.scss (hunk 15 above)
Comment (KarenJoo) [reply src/scss/index.scss:146]: thanks, done. Looks like the img is still a bit stretched no matter how I style it with either of the object-fit values, height/width and rounded-circle

File: feed/index.html (hunk 16 above)
Comment (KarenJoo) [reply feed/index.html]: Ooh, nice! It worked

File: src/scss/index.scss (hunk 7 above)
Comment (KarenJoo) [reply src/scss/index.scss:78]: done

File: src/scss/index.scss (hunk 8 above)
Comment (KarenJoo) [reply src/scss/index.scss]: done

File: src/scss/index.scss (hunk 9 above)
Comment (KarenJoo) [reply src/scss/index.scss:105]: done

File: src/scss/index.scss (hunk 10 above)
Comment (KarenJoo) [reply src/scss/index.scss:107]: done

**Pull request merged**